packages/user-management-widget/src/lib/widget/state/selectors.ts

```diff
@@ -12,9 +13,18 @@ export const getUsersList = createSelector(getRawUsersList, (users) =>
   users.map((user) => ({
     ...user,
     status: userStatusMappings[user.status] || user.status,
+    roles: user.roleNames
+      .concat(user.userTenants?.[0]?.roleNames)
+      .filter(Boolean)
+      .sort()
+      .join(', '),
   })),
 );
 
+export const getTenantRoles = createSelector(getRawTenantRoles, (roles) =>
```

Comment:
I'm not sure if this is the right place to transform the data to match the component structure
I would expect it to happen on the component init fn
WDYT?

Reply:
Moved to component init